connection-service/new-admin-vm/Install-ConnectionServer.ps1

```diff
@@ -917,44 +937,21 @@ isMultiFactorAuthenticate=$isMfa
                 #loop until it's created
                 while(-not $foundCert)
                 {
-                    $cert = $null
                     try {
-                        Write-Host "Looking for LDAPS certificate for $hostname"
-                        $tcpclient = new-object System.Net.Sockets.tcpclient
-                        $tcpclient.Connect($hostname,$port)
-
-                        # Authenticate with SSL - trusting all certificates
-                        $sslstream = new-object System.Net.Security.SslStream -ArgumentList $tcpclient.GetStream(),$false,{$true}
-
-                        $sslstream.AuthenticateAsClient($hostname)
-                        $cert =  [System.Security.Cryptography.X509Certificates.X509Certificate2]($sslstream.remotecertificate)
-                        Write-Host "Found Certificate for $hostname, looking for Issuer Certificate"
-
-                        $chain = New-Object -TypeName System.Security.Cryptography.X509Certificates.X509Chain
-                        # Build the certificate chain from the file certificate
-                        $chain.Build($cert)                        
-                        $listOfCertificates = ($chain.ChainElements | ForEach-Object {$_.Certificate})
-                        
-                        #last one is the root in chain
-                        $caCert=$listOfCertificates[-1]
-                        $content = @(
-                            '-----BEGIN CERTIFICATE-----'
-                            [System.Convert]::ToBase64String($caCert.RawData, 'InsertLineBreaks')
-                            '-----END CERTIFICATE-----'
-                        )
-                       
-                        $content | Out-File -FilePath "$env:systemdrive\$issuerCertFileName" -Encoding ascii
-                        $foundCert=$true
+                        Start-Process C:\OpenSSL-Win64\bin\openssl.exe -ArgumentList "s_client -showcerts -connect ${hostname}:${port}" -RedirectStandardOutput "$env:systemdrive\certInfo.txt"
+                        # Wait a bit to ensure that the previous command completes
```

Comment:
Start-Process -Wait didn't work?

Reply:
It opens a new shell and the new shell waits for you to enter the quit command. I couldn't get it to automatically quit when done